platform/kube/conversion.go

```diff
@@ -53,11 +57,29 @@ func convertLabels(obj meta_v1.ObjectMeta) model.Labels {
 	return out
 }
 
-func convertPort(port v1.ServicePort) *model.Port {
+// Extracts security option for given port from annotation. If there is no such
+// annotation, or the annotation value is not recognized, returns
+// model.SecurityDefault.
+func extractAuthenticationPolicy(port v1.ServicePort, obj meta_v1.ObjectMeta) model.AuthenticationPolicy {
+	if obj.Annotations == nil {
+		return model.AuthenticationDefault
+	}
+	switch obj.Annotations[portAuthenticationAnnotationKey(int(port.Port))] {
+	case "disable":
```

Comment:
can you use some const instead of harcoded strings?

Reply:
This is mapping a text to a constant. Do you think it's easier to follow this way?